Removes the default keyboard bindings

ionutvmi opened the pull request:
I also included some install instructions.

MalcolmK commented:
I'll review it when I have more time! I saw there were quite a lot of changes.

MalcolmK commented:
I haven't tested it, but I trust you did. I do have made some changes to your text. I submitted them as patches to your pull request. You can find them in the pull request section.

ionutvmi commented:
Yep, not much to test, just removed the default key-bindings and updated the text in the readme and install file.
I will go ahead and merge this into develop.
